### PR TITLE
Fix order of CI steps

### PR DESCRIPTION
**Pull request by szepeviktor**
## PR Description

1. Setup PHP + install Composer tool
2. Check out Composer cache
3. All other steps...

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,16 +36,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
 
-      - name: Composer install
-        run: |
-          composer --version
```
Comment (szepeviktor) [inline .github/workflows/ci.yml:42]: 💡 Composer version is displayed in `Setup PHP`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,17 +13,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version: [
-            '7.3',
-            '8.0',
-            '8.1',
-        ]
+        php-version: ['7.3', '8.0', '8.1']
```
Comment (szepeviktor) [inline .github/workflows/ci.yml:16]: This was a YAML problem.

Comment (szepeviktor) [reply .github/workflows/ci.yml:16]: An alternative is

```yaml
php-version:
  - '7.3'
  - '8.0'
  - '8.1'
```

Comment (dereuromark) [reply .github/workflows/ci.yml:16]: I iike the separate lines

Comment (szepeviktor) [reply .github/workflows/ci.yml]: Me too!

**Pull request merged**